corehq/apps/commtrack/tests/util.py

```diff
@@ -217,16 +224,30 @@ def get_single_transfer_block(src_id, dest_id, product_id, quantity, date_string
     ).strip()
 
 
-def fake_sms(user, text):
-    """
-    Fake a commtrack SMS submission for a user.
-    `text` might be "soh myproduct 100"
-    Don't use this with a real user
-    """
-    if not user.phone_number:
-        raise ValueError("User does not have a phone number")
-    if not user.get_verified_number():
-        user.save_verified_number(user.domain, user.phone_number, True, None)
-    domain_obj = Domain.get_by_name(user.domain)
-    parser = StockReportParser(domain_obj, user.get_verified_number())
-    process(user.domain, parser.parse(text.lower()))
+@require_debug_true()
```

Comment:
I made this for using locally, not during tests, to generate stock data for reports.